package.json

```diff
@@ -98,7 +97,7 @@
     "markdown-tables-to-json": "^0.1.7",
     "mocha": "^10.0.0",
     "patch-package": "^8.0.0",
-    "prettier": "3.0.3",
+    "prettier": "^3.0.3",
```

Comment:
Heya @SunMoon97, could you revert this change + the change to the package-lock? Prettier recommends using the exact version so that you make sure everyone is always formatting the files the same way!

Reply:
Done